peakinvestigator/actions/base.py

```diff
@@ -8,11 +8,13 @@
 
 
 from abc import ABCMeta, abstractmethod
+from six import add_metaclass
 import decimal
 
 import simplejson
 
-class BaseAction(metaclass=ABCMeta):
+@add_metaclass(ABCMeta)
+class BaseAction():
```

Comment:
Any ideas why these were added? It looks like those new lines already exist: https://github.com/Veritomyx/PeakInvestigator-Python-SDK/blob/master/peakinvestigator/actions/base.py